core/modules/memman/MemFile.h

```diff
@@ -50,7 +50,7 @@ class MemFile {
     struct MLResult {
         uint64_t bLocked;
         int      retc;
-        MLResult() {}
+        MLResult() : bLocked{false}, retc(0) {}
```

Comment:
I am not sure.  Travis seems like its spinning up two builds per pr right now, and the first always fails.  The second usually works, but in this case did not.  Investigating...